packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -380,6 +380,11 @@ const ReactTestUtils = {
 
   Simulate: null,
   SimulateNative: {},
+
+  interact(callback: () => void) {
+    ReactDOM.unstable_batchedUpdates(callback);
+    ReactDOM.render(null, document.createElement('div'));
```

Comment:
Can we reuse a single node? Maybe put an inline React element instead so we don't bail out.